packages/rocketchat-file-upload/server/methods/sendFileMessage.js

```diff
@@ -24,50 +24,53 @@ Meteor.methods({
 
 		const fileUrl = `/file-upload/${ file._id }/${ encodeURI(file.name) }`;
 
-		const attachment = {
-			title: file.name,
-			type: 'file',
-			description: file.description,
-			title_link: fileUrl,
-			title_link_download: true
-		};
+		(async function() {
```

Comment:
Did you try to remove this `wrapper`? AFAIK all meteor methods are async by default allowing you to call await inside.

Reply:
I had tested a lot of ways to do it, when i tryed to call await without the wrapper it was failing in the build...